launcher/game/updater.rpy

```diff
@@ -71,9 +71,29 @@ init python:
     _("Nightly")
     _("The bleeding edge of Ren'Py development. This may have the latest features, or might not run at all.")
 
+    # Newly added functions. Signed, MPDS
+    # Update: GET OUT OF MY HEAD
 
+    filter_keywords=set()
+    
+    def display_keywords():
+        return "Search phrases:\n"+"\n".join(filter_keywords)
 
-screen update_channel(channels):
+    def add_keyword(temp='monika'):
```

Comment:
the 2 definies I think aren't needed as a regular call to search for stuff is better